services/ui/src/components/Projects/index.js

```diff
@@ -52,7 +52,7 @@ const Projects = ({ projects = [] }) => {
           className="searchInput"
           type="text"
           value={searchInput}
-          onChange={e => setSearchInput(e.target.value)}
+          onChange={e => setSearchInput(e.target.value.replace(/[^0-9a-zA-Z-]/g,''))}
```

Comment:
```suggestion
          onChange={e => setSearchInput(e.target.value.replace(/[^0-9a-zA-Z-.]/g,''))}
```
There may be people who use the search to search for their route URLs, so adding the period separator may be prudent here.

Reply:
The hyphen needs to be last (or first) otherwise it's interpreted as a range, so maybe: `/[^0-9a-zA-z.-]/g`.

Reply:
Great idea - updated now.